go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/aws/aws-sdk-go-v2/service/sqs v1.37.0
 	github.com/aws/aws-sdk-go-v2/service/ssm v1.55.5
 	github.com/awslabs/aws-lambda-go-api-proxy v0.16.2
-	github.com/ethereum/go-ethereum v1.14.13
+	github.com/ethereum/go-ethereum v1.15.11
```

Comment:
humm, can this update be reverted? iirc (I can barely remember) we need to use 1.14.13 specifically for some PDP interactions

Reply:
Yes, I will try, I had a bunch of errors though...